services-geocoding/src/main/java/com/mapbox/api/geocoding/v5/MapboxGeocoding.java

```diff
@@ -251,9 +253,9 @@ public static Builder builder() {
 
     /**
      * Perform a reverse geocode on the provided {@link Point}. Only one point can be passed in as
-     * the query and isn't guaranteed to return a result. If you are an enterprise customer and
-     * wanting to do a batch reverse Geocode, you can use the {@link #query(String)} method
-     * separating them with a semicolon.
+     * the query and isn't guaranteed to return a result. If you
+     * want to do a batch reverse Geocode, you can use the {@link #query(String)} method
+     * separating them with a semicolon. For more information about batch geocoding, * contact <a href="https://www.mapbox.com/contact/sales/">Mapbox sales</a>.
```

Comment:
```suggestion
     * separating them with a semicolon. For more information about batch geocoding, contact <a href="https://www.mapbox.com/contact/sales/">Mapbox sales</a>.
```

Or was the * intentionally left there?